locale/core.yml

```diff
@@ -238,8 +238,11 @@ core:
 
     # These translations are used in the Edit User modal dialog (admin function).
     edit_user:
+      activate: Activate User
       email_label: => core.ref.email
```

Comment:
Seems I made a mistake... There would probabily need to be one `_label` and one `_heading` because one heading and one inputbox/checkbox needs each. So how to deal with this situation?